components/metrics/labels.go

```diff
@@ -0,0 +1,54 @@
+package metrics
+
+import (
+	"context"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type labelKey string
+
+const (
+	labelKeyHandlerName    = "handler_name"
+	labelKeyPublisherName  = "publisher_name"
+	labelKeySubscriberName = "subscriber_name"
+	labelSuccess           = "success"
+	labelAcked             = "acked"
+)
+
+var (
+	labelGetters = map[labelKey]func(context.Context) string{
```

Comment:
looking a bit magic with this init, can it be simpler?

Reply:
Actually, it is not needed anymore, I removed the whole `labelKeys` var e93109c91a43297193a6cc47de0144aceba97944